src/Greeter.js

```diff
@@ -6,15 +6,15 @@ const Greeter = () => {
   let timeOfDay;
 
   if (hours < 12) {
-    timeOfDay = "morning";
+    timeOfDay = "MORNING";
   } else if (hours >= 12 && hours < 17) {
-    timeOfDay = "afternoon";
+    timeOfDay = "AFTERNOON";
   } else {
-    timeOfDay = "night";
+    timeOfDay = "NIGHT";
   }
   return (
     <h1 className="greeter-text">
-      Good {timeOfDay} <br></br> sunshine!
+      GOOD {timeOfDay} <br></br> SUNSHINE!
```

Comment:
Could maybe do "sunshine" for morning and another word for night and afternoon, maybe would look and feel better UX-wise

Reply:
I was thinking of changing it so instead of "sunshine" it greets you with your name instead! :)